keras/optimizers/optimizer_v2/optimizer_v2.py

```diff
@@ -792,8 +793,8 @@ def _set_hyper(self, name, value):
       if (callable(prev_value)
           or isinstance(prev_value,
                         (tf.Tensor, int, float,
-                         learning_rate_schedule.LearningRateSchedule))
-          or isinstance(value, learning_rate_schedule.LearningRateSchedule)):
+                         keras.optimizers.schedules.LearningRateSchedule))
```

Comment:
Fix indent.

Reply:
Line 796 has keras.optimizers.scheduler as the only change , indent has remained the same. Could you please suggest on this ?  I can convert it to : (tf.Tensor, int, float, keras.optimizers.schedules.LearningRateSchedule)) in a single line but that would hamper the preallocated tabs present in the code.